com.github.donadigo.appeditor.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id": "com.github.donadigo.appeditor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.donadigo.appeditor",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "/share/vala",
+        "/man",
+        "*.a",
+        "*.la"
+    ],
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc",
+        "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        "--filesystem=host:ro",
```

Comment:
You'll also want: `--filesystem=~/.local/share/applications:create`

Reply:
Done

Reply:
Sorry I did update my comment to be `xdg-data` rather than `~/.local/share`